### ISSUE TITLE
NullPointerException

### ISSUE THREAD

**Issue by Yonn-Trimoreau** [magento/magento2-phpstorm-plugin#8]
Hello,

PHPStorm is raising an "IDE Fatal Error" on this plugin.

I don't why and how, and the plugin seems to be working fine even if this is raised.

Here is my stack:
```
java.lang.NullPointerException
	at com.magento.idea.magento2plugin.util.VfsUtil.findVfUp(VfsUtil.java:15)
	at com.magento.idea.magento2plugin.util.VfsUtil.findVfUp(VfsUtil.java:21)
	at com.magento.idea.magento2plugin.util.VfsUtil.findVfUp(VfsUtil.java:21)
	at com.magento.idea.magento2plugin.util.VfsUtil.findVfUp(VfsUtil.java:21)
	at com.magento.idea.magento2plugin.util.VfsUtil.findVfUp(VfsUtil.java:21)
	at com.magento.idea.magento2plugin.util.VfsUtil.findVfUp(VfsUtil.java:21)
	at com.magento.idea.magento2plugin.util.VfsUtil.findVfUp(VfsUtil.java:21)
	at com.magento.idea.magento2plugin.util.VfsUtil.findVfUp(VfsUtil.java:21)
	at com.magento.idea.magento2plugin.util.VfsUtil.findVfUp(VfsUtil.java:21)
	at com.magento.idea.magento2plugin.util.VfsUtil.findVfUp(VfsUtil.java:21)
	at com.magento.idea.magento2plugin.util.VfsUtil.findVfUp(VfsUtil.java:21)
	at com.magento.idea.magento2plugin.util.VfsUtil.findVfUp(VfsUtil.java:21)
	at com.magento.idea.magento2plugin.util.VfsUtil.findVfUp(VfsUtil.java:21)
	at com.magento.idea.magento2plugin.completion.provider.FilePathCompletionProvider.findViewVfs(FilePathCompletionProvider.java:105)
	at com.magento.idea.magento2plugin.completion.provider.FilePathCompletionProvider.addCompletions(FilePathCompletionProvider.java:56)
	at com.intellij.codeInsight.completion.CompletionProvider.addCompletionVariants(CompletionProvider.java:36)
	at com.magento.idea.magento2plugin.completion.provider.CompositeCompletionProvider.addCompletions(CompositeCompletionProvider.java:23)
	at com.intellij.codeInsight.completion.CompletionProvider.addCompletionVariants(CompletionProvider.java:36)
	at com.intellij.codeInsight.completion.CompletionContributor.fillCompletionVariants(CompletionContributor.java:152)
	at com.intellij.codeInsight.completion.CompletionService.getVariantsFromContributors(CompletionService.java:83)
	at com.intellij.codeInsight.completion.CompletionResultSet.runRemainingContributors(CompletionResultSet.java:149)
	at com.intellij.codeInsight.completion.CompletionResultSet.runRemainingContributors(CompletionResultSet.java:142)
	at com.intellij.codeInsight.completion.CompletionResultSet.runRemainingContributors(CompletionResultSet.java:132)
	at com.intellij.codeInsight.completion.XmlNoVariantsDelegator.fillCompletionVariants(XmlNoVariantsDelegator.java:28)
	at com.intellij.codeInsight.completion.CompletionService.getVariantsFromContributors(CompletionService.java:83)
	at com.intellij.codeInsight.completion.CompletionResultSet.runRemainingContributors(CompletionResultSet.java:149)
	at com.intellij.codeInsight.completion.CompletionResultSet.runRemainingContributors(CompletionResultSet.java:142)
	at com.intellij.codeInsight.template.impl.LiveTemplateCompletionContributor$2.addCompletions(LiveTemplateCompletionContributor.java:90)
	at com.intellij.codeInsight.completion.CompletionProvider.addCompletionVariants(CompletionProvider.java:36)
	at com.intellij.codeInsight.completion.CompletionContributor.fillCompletionVariants(CompletionContributor.java:152)
	at com.intellij.codeInsight.completion.CompletionService.getVariantsFromContributors(CompletionService.java:83)
	at com.intellij.codeInsight.completion.CompletionService.performCompletion(CompletionService.java:112)
	at com.intellij.codeInsight.completion.CompletionProgressIndicator.a(CompletionProgressIndicator.java:839)
	at com.intellij.codeInsight.completion.CompletionProgressIndicator.access$600(CompletionProgressIndicator.java:92)
	at com.intellij.codeInsight.completion.CompletionProgressIndicator$1CalculateItems.run(CompletionProgressIndicator.java:821)
	at com.intellij.openapi.application.impl.ApplicationImpl.tryRunReadAction(ApplicationImpl.java:1127)
	at com.intellij.codeInsight.completion.AsyncCompletion.tryReadOrCancel(CompletionThreading.java:181)
	at com.intellij.codeInsight.completion.CompletionProgressIndicator.b(CompletionProgressIndicator.java:832)
	at com.intellij.codeInsight.completion.AsyncCompletion.a(CompletionThreading.java:108)
	at com.intellij.openapi.progress.impl.CoreProgressManager.a(CoreProgressManager.java:157)
	at com.intellij.openapi.progress.impl.CoreProgressManager.a(CoreProgressManager.java:534)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:479)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:94)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:144)
	at com.intellij.codeInsight.completion.AsyncCompletion.a(CompletionThreading.java:104)
	at com.intellij.openapi.application.impl.ApplicationImpl$1.run(ApplicationImpl.java:315)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
```
Just in case it lights up something in someone's head...

**Comment by voleye** [comment]
@Yonn-Trimoreau  thank you for this report the issue was fixed at PR #11

**Comment by Vinai** [comment]
Still happening in 2019-02-19:

```
java.lang.NullPointerException
	at com.magento.idea.magento2plugin.util.VfsUtil.findVfUp(VfsUtil.java:15)
	at com.magento.idea.magento2plugin.util.VfsUtil.findVfUp(VfsUtil.java:21)
	at com.magento.idea.magento2plugin.util.VfsUtil.findVfUp(VfsUtil.java:21)
	at com.magento.idea.magento2plugin.util.VfsUtil.findVfUp(VfsUtil.java:21)
	at com.magento.idea.magento2plugin.util.VfsUtil.findVfUp(VfsUtil.java:21)
	at com.magento.idea.magento2plugin.util.VfsUtil.findVfUp(VfsUtil.java:21)
	at com.magento.idea.magento2plugin.util.VfsUtil.findVfUp(VfsUtil.java:21)
	at com.magento.idea.magento2plugin.util.VfsUtil.findVfUp(VfsUtil.java:21)
	at com.magento.idea.magento2plugin.util.VfsUtil.findVfUp(VfsUtil.java:21)
	at com.magento.idea.magento2plugin.util.VfsUtil.findVfUp(VfsUtil.java:21)
	at com.magento.idea.magento2plugin.completion.provider.FilePathCompletionProvider.findViewVfs(FilePathCompletionProvider.java:105)
	at com.magento.idea.magento2plugin.completion.provider.FilePathCompletionProvider.addCompletions(FilePathCompletionProvider.java:56)
	at com.intellij.codeInsight.completion.CompletionProvider.addCompletionVariants(CompletionProvider.java:36)
	at com.magento.idea.magento2plugin.completion.provider.CompositeCompletionProvider.addCompletions(CompositeCompletionProvider.java:23)
	at com.intellij.codeInsight.completion.CompletionProvider.addCompletionVariants(CompletionProvider.java:36)
	at com.intellij.codeInsight.completion.CompletionContributor.fillCompletionVariants(CompletionContributor.java:141)
	at com.intellij.codeInsight.completion.CompletionService.getVariantsFromContributors(CompletionService.java:87)
	at com.intellij.codeInsight.completion.CompletionResultSet.runRemainingContributors(CompletionResultSet.java:149)
	at com.intellij.codeInsight.completion.CompletionResultSet.runRemainingContributors(CompletionResultSet.java:142)
	at com.intellij.codeInsight.completion.CompletionResultSet.runRemainingContributors(CompletionResultSet.java:132)
	at com.intellij.codeInsight.completion.XmlNoVariantsDelegator.fillCompletionVariants(XmlNoVariantsDelegator.java:27)
	at com.intellij.codeInsight.completion.CompletionService.getVariantsFromContributors(CompletionService.java:87)
	at com.intellij.codeInsight.completion.CompletionResultSet.runRemainingContributors(CompletionResultSet.java:149)
	at com.intellij.codeInsight.completion.CompletionResultSet.runRemainingContributors(CompletionResultSet.java:142)
	at com.intellij.codeInsight.completion.XmlCompletionContributor$3.addCompletions(XmlCompletionContributor.java:126)
	at com.intellij.codeInsight.completion.CompletionProvider.addCompletionVariants(CompletionProvider.java:36)
	at com.intellij.codeInsight.completion.CompletionContributor.fillCompletionVariants(CompletionContributor.java:141)
	at com.intellij.codeInsight.completion.XmlCompletionContributor.fillCompletionVariants(XmlCompletionContributor.java:181)
	at com.intellij.codeInsight.completion.CompletionService.getVariantsFromContributors(CompletionService.java:87)
	at com.intellij.codeInsight.completion.CompletionResultSet.runRemainingContributors(CompletionResultSet.java:149)
	at com.intellij.codeInsight.completion.CompletionResultSet.runRemainingContributors(CompletionResultSet.java:142)
	at com.intellij.codeInsight.template.impl.LiveTemplateCompletionContributor$2.addCompletions(LiveTemplateCompletionContributor.java:96)
	at com.intellij.codeInsight.completion.CompletionProvider.addCompletionVariants(CompletionProvider.java:36)
	at com.intellij.codeInsight.completion.CompletionContributor.fillCompletionVariants(CompletionContributor.java:141)
	at com.intellij.codeInsight.completion.CompletionService.getVariantsFromContributors(CompletionService.java:87)
	at com.intellij.codeInsight.completion.impl.CompletionServiceImpl.performCompletion(CompletionServiceImpl.java:63)
	at com.intellij.codeInsight.completion.CompletionProgressIndicator.calculateItems(CompletionProgressIndicator.java:827)
	at com.intellij.codeInsight.completion.CompletionProgressIndicator.runContributors(CompletionProgressIndicator.java:812)
	at com.intellij.codeInsight.completion.CodeCompletionHandlerBase.lambda$null$2(CodeCompletionHandlerBase.java:272)
	at com.intellij.openapi.application.impl.ApplicationImpl.tryRunReadAction(ApplicationImpl.java:1168)
	at com.intellij.codeInsight.completion.AsyncCompletion.tryReadOrCancel(CompletionThreading.java:169)
	at com.intellij.codeInsight.completion.CodeCompletionHandlerBase.lambda$doComplete$3(CodeCompletionHandlerBase.java:269)
	at com.intellij.codeInsight.completion.AsyncCompletion.lambda$null$0(CompletionThreading.java:95)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcess$2(CoreProgressManager.java:164)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:582)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:532)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:87)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:151)
	at com.intellij.codeInsight.completion.AsyncCompletion.lambda$startThread$1(CompletionThreading.java:91)
	at com.intellij.openapi.application.impl.ApplicationImpl$1.run(ApplicationImpl.java:314)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
```

**Comment by paliarush** [comment]
The fix will be released in 0.3.0 version in the nearest days.
